charts/redpanda/templates/statefulset.yaml

```diff
@@ -389,6 +389,27 @@ spec:
             {{- end }}
           {{- end }}
       {{- end }}
+      {{- if and .Values.rbac.enabled .Values.statefulset.sideCars.controllers.enabled }}
+        - name: redpanda-controllers
+          image: {{ .Values.statefulset.sideCars.controllers.image.repository }}:{{ .Values.statefulset.sideCars.controllers.image.tag }}
+          command:
+            - /manager
+          args:
+            - --operator-mode=false
+            - --namespace={{ .Release.Namespace }}
+            - --health-probe-bind-address=:8085
+            - --metrics-bind-address=:9082
```

Comment:
Can you move those ports to values.yaml?

They should not collide with Redpanda ports.

Reply:
This is why they were chosen like this, yeah will move it to values.